core/src/main/scala/kafka/tools/ReplicaVerificationTool.scala

```diff
@@ -116,7 +121,13 @@ object ReplicaVerificationTool extends Logging {
     if (options.has(versionOpt)) {
       CommandLineUtils.printVersionAndDie()
     }
-    CommandLineUtils.checkRequiredArgs(parser, options, brokerListOpt)
+
+    val effectiveBrokerListOpt = if (options.has(bootstrapServerOpt))
+      bootstrapServerOpt
+    else
+      brokerListOpt
```

Comment:
Is it possible that user provided both `bootstrapServerOpt` and `brokerListOpt` at the same time?

Reply:
Yes. But `bootstrapServerOpt` has the preference.

Reply:
Fixed. See `ReplicaVerificationToolOptions#effectiveBrokerListOpt`.